### PR TITLE
feat(gke): update guide for KFD 1.24

### PR DESCRIPTION
**Pull request by ralgozino**
Update GKE getting started guide to support KFD 1.24.0

### CODE REVIEW

File: fury-on-gke/README.md [hunk 1]
```diff
@@ -379,88 +386,82 @@ furyctl vendor -H
...
+- ../vendor/katalog/monitoring/prometheus-operated
 - ../vendor/katalog/monitoring/grafana
-- ../vendor/katalog/monitoring/kube-proxy-metrics
 - ../vendor/katalog/monitoring/kube-state-metrics
 - ../vendor/katalog/monitoring/node-exporter
-- ../vendor/katalog/monitoring/prometheus-operated
-- ../vendor/katalog/monitoring/prometheus-operator
+- ../vendor/katalog/monitoring/alertmanager-operated
+- ../vendor/katalog/monitoring/kube-proxy-metrics
+- ../vendor/katalog/monitoring/eks-sm
```
Comment (nutellinoit) [inline fury-on-gke/README.md:441]: I think we need to use the gke-sm one here

File: fury-on-gke/manifests/kustomization.yaml [hunk 2]
```diff
@@ -6,21 +10,22 @@ resources:
...
+- ../vendor/katalog/monitoring/prometheus-operated
 - ../vendor/katalog/monitoring/grafana
-- ../vendor/katalog/monitoring/kube-proxy-metrics
 - ../vendor/katalog/monitoring/kube-state-metrics
 - ../vendor/katalog/monitoring/node-exporter
-- ../vendor/katalog/monitoring/prometheus-operated
-- ../vendor/katalog/monitoring/prometheus-operator
+- ../vendor/katalog/monitoring/alertmanager-operated
+- ../vendor/katalog/monitoring/kube-proxy-metrics
+- ../vendor/katalog/monitoring/eks-sm
```
Comment (nutellinoit) [inline fury-on-gke/manifests/kustomization.yaml:28]: gke-sm here also

**Review by nutellinoit — CHANGES_REQUESTED** [review]
Minor changes, overall LGTM!

File: fury-on-gke/manifests/kustomization.yaml (hunk 2 above)
Comment (ralgozino) [reply fury-on-gke/manifests/kustomization.yaml:28]: :facepalm: thanks!

File: fury-on-gke/README.md (hunk 1 above)
Comment (ralgozino) [reply fury-on-gke/README.md:441]: :facepalm: thanks!

**Review by nutellinoit — APPROVED** [review]
LGTM

**Pull request merged**